tests/testProvenanceSetup.py

```diff
@@ -132,15 +136,15 @@ def testAddCmd(self):
         self.assertEquals(cmds[0][0], "recProv.py")
         self.assertEquals(cmds[0][1], "-v")
         self.assertEquals(cmds[0][2], "4")
-        
+
         self.setup.addWorkflowRecordCmd("dbingest.py", "-u ray".split(),
                                         "/usr/local/bin/dbingest.py")
```

Comment:
I think the question is "is it reasonable to add a command to /usr/local/bin"? I would expect commands in ctrl_provenance to add commands to `os.path.join(getPackageDir("ctrl_provenance"), "bin")` if the command needs any software managed by eups. (It also seems odd to add a command and not verify that it was added, or at least that the file exists, but that's another story).